src/app/modules/shared/layout/footer/footer.component.ts

```diff
@@ -7,5 +7,5 @@ import { Component } from '@angular/core';
   styleUrls: ['./footer.component.scss']
 })
 export class FooterComponent {
-  version?: string = '1.0.1';
+  version?: string = '1.0.2';
```

Comment:
interesting.
now that you are setting this also in package.json, I am wondering if we could use a pre-publish step to modify the version number, import it into a config, and express it in places like this dynamically